Project.toml

```diff
@@ -20,6 +20,7 @@ OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Ah this seems like leftovers from when we accidently installed Revise.jl in the Oceananigans environment haha.

We probably want to remove it before merging with `]rm Revise`.

Reply:
Sorry about that.  I have since removed it and then pushed the changes.  All the tests pass on my comptuer and hope they will pass on github as well.